extensions/amp-sticky-ad/1.0/validator-amp-sticky-ad.protoascii

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+tags: {  # <script custom-element="amp-sticky-ad">
+  html_format: AMP  # Ads are not allowed inside ads.
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-sticky-ad"
+    allowed_versions: "0.1"
+    allowed_versions: "1.0"
+    allowed_versions: "latest"
+    deprecated_versions: "0.1"
+    requires_usage: DEPRECATED_WARNING
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-sticky-ad>
+  html_format: AMP
+  tag_name: "AMP-STICKY-AD"
+  disallowed_ancestor: "AMP-APP-BANNER"
+  disallowed_ancestor: "AMP-SIDEBAR"
+  requires_extension: "amp-sticky-ad"
```

Comment:
Aside, ignore for review: @Gregable we should think about how this handles versioning for extension scripts.